lib/Query/Resolver.php

```diff
@@ -298,7 +298,9 @@ protected function addForeignKeys(Table $table)
                     continue;
                 }
 
-                $foreignTable = $relation->mapper()->getMapper($relation->entityName())->table();
+                $foreignTableMapper = $relation->mapper()->getMapper($relation->entityName());
+                $foreignTableMapper->migrate();
```

Comment:
Thank you for your feedback. If you reference a non-existent table it will cause an error during the migration. That's the reason behind this modification. I'll create a test and fix for your case. Could you please give me details about your entity?